contributors/devel/sig-architecture/validation-suites.md

```diff
@@ -0,0 +1,92 @@
+# Validation Suite Testing in Kubernetes
+
+A Kubernetes Validation suite is a subset of e2e tests that are used to
+validate functionality of an optional capability of Kubernetes for use
+with a specific release. Validation suites can grow or shrink over time, and can also
+change as a functionality progresses from alpha, to beta, and to GA. The flexibility
+of Validation suites to change based on experimentation and feedback is a key 
+characteristic that differentiates Validation suites from 
+[Conformance Tests](../sig-architecture/conformance-tests.md) and 
+[Conformance profiles](https://github.com/cncf/k8s-conformance/blob/master/terms-conditions/Certified_Kubernetes_Terms.md). In addition, Validation suites differ from  
+Conformance profiles in that they do not require conformance certification approval
+in order to be created and utilized. As a result, Validation 
+suites can be created to validate the functionality of an optional 
+capability and the more lengthy community discussion of whether the validation suite
+should be promoted to become a Conformance profile or merged into the base
+Conformance Tests can be deferred.  Validation suites allow the Kuberentes e2e
+testing team to rapidly create test suites for a variety of use cases.  They may
+also be used to identify a group of tests that validate functionality on an
+optional platform such as Kuberenetes on Windows.  Kubernetes SIG groups are
```

Comment:
@bradtopol I think validation suites should be targeted to functionality but not at platforms. Can you please remove this sentence with Windows as an example.

Reply:
Will do!